IntegrationTests/tests_04_performance/test_01_resources/test_1000_tcpconnections.swift

```diff
@@ -29,6 +29,19 @@ fileprivate final class ReceiveAndCloseHandler: ChannelInboundHandler {
     }
 }
 
+fileprivate final class LingerSettingHandler: ChannelInboundHandler {
+    typealias InboundIn = ByteBuffer
+    typealias OutboundOut = ByteBuffer
+    
+    public func handlerAdded(context: ChannelHandlerContext) {
+        _ = (context.channel as? SocketOptionProvider)?.setSoLinger(linger(l_onoff: 1, l_linger: 0))
```

Comment:
shouldn't we blow up in a `.whenFailure { ... }` here?

Reply:
Can't hurt. @PeterAdams-A as you've adopted this branch, want to keep going?

Reply:
Done this.